src/systemTest/java/com/flightstats/hub/system/functional/TimedWebhooksTest.java

```diff
@@ -0,0 +1,131 @@
+package com.flightstats.hub.system.functional;
+
+import com.flightstats.hub.model.Webhook;
+import com.flightstats.hub.model.WebhookType;
+import com.flightstats.hub.system.config.DependencyInjector;
+import com.flightstats.hub.system.service.CallbackService;
+import com.flightstats.hub.system.service.ChannelService;
+import com.flightstats.hub.system.service.WebhookService;
+import com.flightstats.hub.utility.StringHelper;
+import lombok.extern.slf4j.Slf4j;
+import org.awaitility.Awaitility;
+import org.awaitility.Duration;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+@Slf4j
+class TimedWebhooksTest extends DependencyInjector {
```

Comment:
This class has a lot of helper-type methods. Probably not too important, but are there any that are valuable enough to extract to a shared location?

Reply:
moved one, but the others are fairly specific